stacker/config/translators/kms.py

```diff
@@ -1,7 +1,27 @@
 import base64
+import os
+
 import botocore.session
 
 
+def _get_config_directory():
```

Comment:
Both of these helper functions seem useful/generic enough that I think they should probably be in the base translators module - I could see other translators, in the future, wanting to use them.